src/state/session/agent.ts

```diff
@@ -204,6 +205,7 @@ export function agentToSessionAccount(
     active: agent.session.active,
     status: agent.session.status as SessionAccount['status'],
     pdsUrl: agent.pdsUrl?.toString(),
+    isSelfHosted: !agent.serviceUrl.toString().startsWith(BSKY_SERVICE),
```

Comment:
The current value of `BSKY_SERVICE` is `https://bsky.social`. So services at `https://bsky.social.example.com` or even `https://bsky.socialblade.com` would be erroneously classified as not-self-hosted. Not sure if this matters much in practice.

Reply:
Yeahhh kinda just chucked this in as a temp thing. I've since connected with Matthieu and have a ticket on the todo list to make this stable before we start relying on this value.